.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 16.x, 14.x ]
```

Comment:
I'd be fine with omitting 14.x from this list, we should be using at least the LTS version, which is 16. I'll add a card to ensure devs are using the right node version.